test/special/mocha.js

```diff
@@ -1,47 +1,61 @@
 import 'should';
 import fs from 'fs';
 import path from 'path';
-import parse from '../../src/special/mocha';
+import parser from '../../src/special/mocha';
 import { clearCache } from '../../src/utils/get-scripts';
+import {
+  getTestParserWithTempFile,
+  getTestParserWithContentPromise,
+} from '../utils';
+
+const testParser = getTestParserWithContentPromise(parser);
+const testParserTempFile = getTestParserWithTempFile(parser);
```

Comment:
We mix the 2 different strategies because `require` is used in some cases and in others not.